src/maliput_malidrive/builder/road_geometry_builder.cc

```diff
@@ -283,9 +283,11 @@ RoadGeometryBuilder::LaneConstructionResult RoadGeometryBuilder::BuildLane(
                              allow_semantic_errors /* allow_negative_width */);
   // When semantic errors aren't allowed G1 contiguity must be enforced for all lanes.
   // Otherwise, only drivable lanes are enforced.
-  const bool enforce_contiguity = !allow_semantic_errors || is_driveable_lane(*lane);
+  const bool enforce_contiguity_non_driveable_lanes = !allow_semantic_errors || is_driveable_lane(*lane);
+  const bool adapt_lane_widths = allow_semantic_errors && is_driveable_lane(*lane);
```

Comment:
Add a small doc here mentioning what is the logic, and also remember to add docs to the params.h file.